CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    public Jester(IJokeOutput jokeOutput, IJokeService jokeService)
+    {
+        JokeOutput = jokeOutput ?? throw new ArgumentNullException(nameof(jokeOutput));
+        JokeService = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
+    }
+
+    public IJokeOutput JokeOutput { get; }
+    public IJokeService JokeService { get; }
```

Comment:
```suggestion
    private IJokeOutput _JokeOutput;
    private IJokeService _JokeService;
```
If these don't need to be public, make them private. Right now, they are only being used within this class.